.circleci/config.yml

```diff
@@ -188,10 +232,15 @@ workflows:
           filters:
             tags:
               only: /v[0-9]+(\.[0-9]+)*/
+      - build-prof-test:
+          filters:
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
```

Comment:
Running the memory testing only on a release tag might be too late. By that point won't our build system already have built new binaries?

If the memory test doesn't take too long we should probably have it run all the time on CI to immediately catch any regression.

Reply:
It's not descriptive but that config means that on tag push circle will also run the `build-prof-test` job(if `filters` is not added the job will be skipped when releasing), see the CircleCI v2 docs on [that](https://circleci.com/docs/2.0/configuration-reference/#tags).

So the job will run when doing any branch push, you can also see [here](https://circleci.com/gh/steve-chavez/postgrest/590?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) that that job ran on the latest commit on this branch.